crates/signs/src/bars.rs

```diff
@@ -182,6 +193,15 @@ impl Material for BarMaterial {
 #[derive(Component)]
 struct BarChild(Entity);
 
+#[derive(Component)]
+struct BarUpdateTimer(Timer);
+
+impl Default for BarUpdateTimer {
+    fn default() -> Self {
+        Self(Timer::new(UPDATE_VISIBILITY_DURATION, TimerMode::Once))
```

Comment:
I've realized during the testing that this implementation leads to all bars being briefly displayed after entity spawning. One hacky way to solve this is to do a variation of the following:

```rust
let mut timer = Timer::new(UPDATE_VISIBILITY_DURATION, TimerMode::Once);
// Avoid triggering visibility after spawning.
timer.tick(UPDATE_VISIBILITY_DURATION);
Self(timer)
```